browser/src/canvas/sections/CommentSection.ts

```diff
@@ -1115,6 +1114,14 @@ export class Comment extends CanvasSectionObject {
 
 	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 	public onLostFocus (e: any): void {
+
+		if (!this.isEdit() || this.sectionProperties.container.contains(e.relatedTarget))
+			return;
+		if (this.sectionProperties.nodeReply.contains(e.srcElement)) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement
`Deprecated: This feature is no longer recommended. Though some browsers might still support it, it may have already been removed from the relevant web standards`

is that the same as `e.relatedTarget` ?

Reply:
can we add comment above this function documenting why, when it is called and what it does ?
so next person will not have to read the wall of text

Reply:
and for `onLostFocusReply` the same

Reply:
or better: in both onLostFocus and onLostFocusReply move duplicated chunk of code which is  hard to read - so it will get name and explain itself :)